src/components/pages/course-view/CourseViewContainer.jsx

```diff
@@ -1,56 +1,50 @@
 import React, { useState } from 'react';
 import { Card } from 'antd';
+import { ModuleView } from '../index';
```

Comment:
fun fact: `index.js` files are great because you don't need to name them in the import statement. This line could be replaced with `import {ModuleView} from "../"`

Reply:
note: since this import just moves up a level, I think naming the `index` actually IS useful to the reader, but if you didn't know about the index pattern I wanted to point it out.

Reply:
I actually forgot that index isn't needed in this path so great point Chaz! I'm down for adjusting however we see fit as a team to keep that standard going as well.